src/ConnectedKubernetes/ConnectedKubernetes.Autorest/docs/Set-AzConnectedKubernetes.md

```diff
@@ -98,7 +126,74 @@ Location Name              ResourceGroupName
 eastus   azps_test_cluster azps_test_group
 ```
 
-This command disable gateway feature of a connected kubernetes.
+This command enables gateway feature of a connected kubernetes cluster.
+
+### Example 3: Enable gateway for a connected Kubernetes cluster with InputObject
+```powershell
+# Get an existing cluster first
+$inputObject = Get-AzConnectedKubernetes -ClusterName azps_test_cluster -ResourceGroupName azps_test_group -SubscriptionId $subscriptionId
+# Enable gateway and set gateway resource Id 
+$outputObject.GatewayEnabled=$true
+$outputObject.GatewayResourceId=$gatewayResourceId
+Set-AzConnectedKubernetes -InputObject $inputObject     
```

Comment:
mix of `$outputObject` and `$inputObject`

Reply:
nice catch!